packages/fxa-settings/src/lib/hooks.tsx

```diff
@@ -109,8 +113,8 @@ export function useAlertBar({
   );
 
   const error = useCallback(
-    (message: ReactNode) => {
-      setContent(message);
+    (message: ReactNode, error?: ApolloError) => {
```

Comment:
I think you could also allow the `AuthenticationError` here, because it does have a `message` on it. Then you wouldn't have to remember to cast the type in those few spots above.